src/topp/IDFileConverter.cpp

```diff
@@ -329,6 +330,20 @@ class TOPPIDFileConverter :
       else if (in_type == FileTypes::IDXML)
       {
         IdXMLFile().load(in, protein_identifications, peptide_identifications);
+        // get spectrum_references from the mz data, if necessary:
+        if (!mz_file.empty())
```

Comment:
Coming back to this: I think the comment got pinned on the `if (...)` but was meant for the content/implications of that condition, which are now analogous to how `addMissingRTs...` is handled. 
Can we close the review?